privatebin/templates/statefulset.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
 spec:
+  serviceName: {{ include "privatebin.fullname" . }}
```

Comment:
this is now already in master

```suggestion
```

Reply:
Hello @bdashrad no problem, will do in ~12 hours.
P.S. now I am sorry for the delay.